packages/services/src/services/VideoPlayer/VideoPlayer.js

```diff
@@ -226,6 +237,17 @@ class VideoPlayerAPI {
             resolve(kdp);
           },
         });
+
+        if (isCustomCreated) {
+          const previousVideoDiv = document
```

Comment:
At this point, does `document.querySelector('dds-tabs-extended-media').shadowRoot.querySelector(targetId)` exist yet? Otherwise if this works we can stick with this for now since this is only a temporary measure.